inventory/routes/concepts/inter_warehouse.rst

```diff
@@ -1,131 +1,128 @@
    :align: center
+   :alt: View of the inventory adjustment form before hitting the start button.
 
-Next, click on **Start Inventory**. A new window will open where you will
-be able to input your existing products. Select add an item and indicate
-the **Real Quantity** available in the warehouse. The theoretical quantity
-can not be changed as it represents a computed quantity from purchase
-and sales orders.
+Next, click on **Start Inventory**. You will be able to input your existing products in the new
+window. Select add an item and indicate the **Real Quantity** available in the warehouse.
+The theoretical quantity can not be changed as it represents a computed one from purchase and sales
+orders.
 
-.. image:: media/inter05.png
+.. image:: inter_warehouse/in-progress-annual-inventory.png
    :align: center
+   :alt: View of the actual physical inventory of the new warehouse.
 
-Don't forget to validate your inventory once you have recorder the state of
-all yours product.
+.. tip::
+   Don't forget to validate your inventory once you have recorded the state of all your products.
 
 Create an internal transfer
 ===========================
 
-The final step is to create your internal transfer. If you want to
-tranfer 2 units of a product from your first warehouse to another one in
-Brussels, proceed as follows:
+The final step is to create your internal transfer. If you want to transfer 8 units of a product
+from your first warehouse to another one, proceed as follows:
 
-From your dashboard, select a internal movement of one of the two
-warehouses. To do so, click on :menuselection:`More  --> Transfer`.
+From your dashboard, select a internal movement of one of the two warehouses. To do so, click on
+:menuselection:`More  --> Planned Transfer`.
 
-.. image:: media/inter06.png
+.. image:: inter_warehouse/internal-transfer-choice.png
    :align: center
+   :alt: View of the choice between planned transfer and immediate transfer.
 
-A new window will open where you will be able to select the source
-location zone (in this case our "old warehouse") and the destination
-location zone (in this case our "new" warehouse located in Brussels).
+On the new window, select the source location zone (in this case your "old warehouse") and the
+destination location zone (in this case your "new" warehouse).
 
-Add the products you want to transfer by clicking on **Add an Item** and
-don't forget to **Validate** or **Mark as TODO** once you are done.
+Add the products you want to transfer by clicking on **Add an Item** and don't forget to
+**Save** and **Mark as TODO** once you are done.
 
-.. image:: media/inter07.png
+.. image:: inter_warehouse/planned-internal-transfer.png
    :align: center
+   :alt: View of the internal transfer form.
 
-If you select **Validate**, Odoo will process all quantities to transfer.
+If you selected **Immediate Transfer**, Odoo would have processed the transfer automatically after
+your click on **Validate**.
```

Comment:
grammar: I believe the tenses don't follow well.
- If you selected --> Odoo _[simple present]_ processes the transfer
- - If you _[past perfect]_ had selected --> Odoo would have processed

```suggestion
If you selected **Immediate Transfer**, Odoo processes the transfer automatically after you clicked 
on **Validate**.
```

Reply:
Yup, I should have reread myself one more time x)